### PR TITLE
Added some references to bibliography

### PR DESCRIPTION
**Pull request by volkm**
- Updated information on Storm STTT paper (also see https://github.com/moves-rwth/stormpy/issues/97)
- Added some own papers using Storm as backend

### CODE REVIEW

File: _bibliography/references.bib [hunk 1]
```diff
@@ -28,37 +28,112 @@
...
+  pages     = {713--732},
+  publisher = {Springer},
+  year      = {2022},
+  doi       = {10.1007/978-3-031-06773-0\_38},
+  arxiv-id  = {2202.02829},
+  material  = {https://doi.org/10.5281/zenodo.5834212},
 }
 
-@article{HJKQV21,
+@article{HJKQV22,
```
Comment (tquatmann) [inline _bibliography/references.bib:70]: This key is used [here](https://github.com/moves-rwth/storm/blob/d4b7e896cddcf00e98705ca58376d27db60327a7/benchmarks.md?plain=1#L26). Could you update this?

**Comment by tquatmann** [comment]
LGTM! (except that one comment)
Thanks for updating the list :)

File: _bibliography/references.bib (hunk 1 above)
Comment (volkm) [reply _bibliography/references.bib:70]: Done.

**Comment by volkm** [comment]
Thanks for the hint. I fixed the reference.

**Pull request merged**